src/model/game/StorePlatform.ts

```diff
@@ -7,3 +7,15 @@ export enum StorePlatform {
     XBOX_GAME_PASS = "Xbox Game Pass",
     OTHER = "Other",
 }
+
+export function GetStorePlatformFromName(name: string): StorePlatform {
+    const platformNames = Object.keys(StorePlatform);
```

Comment:
Nitpick: Don't know if it makes much difference in practice, but I find it somewhat cleaner to do this as 
```typescript
const platformNames = Object.keys(StorePlatform) as (keyof typeof StorePlatform)[];
```
and then later use
```typescript
return StorePlatform[platformName];
```